index.html

```diff
@@ -107,8 +107,8 @@ <h3 class="sign-up__title">
                 Chcesz zwiedzić z nami Trójmiasto? <br>Zostaw nam maila a poinformujemy Cię, gdy pojawią się nowe funkcje.
             </h3>
             <form class="sign-up__form">
-                <input class="form__input" type="email" name="input" placeholder="Tu wpisz maila" required value="">
-                <input class="form__button" type="submit" value="ok">
+                <input class="form__input" type="email" name="input" placeholder="Tu wpisz adres e-maila" required value="">
```

Comment:
może być "e-mail" nie trzeba odmieniać :D

Reply:
strasznie to wygląda!!!! do zmiany !